independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/providers/serialisers/InputStreamMessageBodyHandler.java

```diff
@@ -26,9 +26,17 @@ public boolean isWriteable(Class<?> type, Type genericType, Annotation[] annotat
         return InputStream.class.isAssignableFrom(type);
     }
 
+    protected boolean doIsWriteable(Class<?> type) {
+        return InputStream.class.isAssignableFrom(type);
+    }
+
     @Override
     public void writeTo(InputStream inputStream, Class<?> type, Type genericType, Annotation[] annotations, MediaType mediaType,
             MultivaluedMap<String, Object> httpHeaders, OutputStream entityStream) throws IOException, WebApplicationException {
+        doIsWriteable(inputStream, entityStream);
+    }
+
+    protected void doIsWriteable(InputStream inputStream, OutputStream entityStream) throws IOException {
```

Comment:
This method name doesn't sound right. How about `writeTo(InputStream,OutputStream)`?